lib/puppet/type.rb

```diff
@@ -1638,6 +1638,14 @@ class { 'apt-updates':
     }
   end
 
+  newmetaparam(:export, :parent => RelationshipMetaparam, :attributes => {:direction => :out, :events => :NONE}) do
+    desc "Control which produced capabilities are actually instantiated"
```

Comment:
this is a little skimpy - this text is what will appear as the on-line documentation. Most others have around 10-15 lines of text and include an example.